assets/scss/core/_base.scss

```diff
@@ -31,9 +29,9 @@ label {
   border-color: $dp-color-lightgrey transparent transparent transparent;
   position: absolute;
   right: 15px;
-  top: 67%;
+  top: 2.5em;
```

Comment:
define in ems depends of the parent value right? it's not a problem this? in any case?

Reply:
We are talking about an arrow inside a dropdown, it should depend on the parent always, at least that's my opinion, of course i can be wrong about this.

Reply:
@GusBaamonde this change is still pending, right?

Feel free to call @mvillaseco if it is not clear.